cups/templates/printers.conf.j2

```diff
@@ -0,0 +1,22 @@
+# Printer configuration file for CUPS v1.7.2
+# Written by cupsd
+# DO NOT EDIT THIS FILE WHEN CUPSD IS RUNNING
+<DefaultPrinter contest>
+UUID urn:uuid:db002b1c-c421-3c4b-46da-d0e7d4d2ebb4
+AuthInfoRequired none
+Info contest
+DeviceURI ipp://printsvr:631/classes/contest
+PPDTimeStamp *
+State Idle
+StateTime 1463533936
+Type 8388614
+Accepting Yes
+Shared No
+ColorManaged Yes
+JobSheets {{ job_sheet}} none
```

Comment:
```suggestion
JobSheets {{ job_sheet }} none
```
The convention is to have spaces there,

You could try to run ansible-lint on the role for these conventions, most of the time I can agree with these conventions.